### DISCUSSION TITLE
Export Alerts

### DISCUSSION THREAD

**Discussion by b4b857f6ee** [Security-Onion-Solutions/securityonion#5693] [Unsupported Versions]
Hello :),

I'm on 2.3.50 and i'm trying to configure to export as syslog the alert dataset.

I try this but it's not working :'(....

https://docs.securityonion.net/en/2.3/logstash.html#forwarding-events-to-an-external-destination

Any tips?

Thank you for your help!

**Comment by b4b857f6ee** [comment]
My configuration  (for tcp here)

![image](https://user-images.githubusercontent.com/36012515/135135776-ed5f69f9-0876-425b-902f-091757dfdd44.png)

But after few test now... i got no event anymore on the kibana lol....

**Reply by weslambert** [reply]
Have you tried using something like tcpdump from the receiver side to see if events are even hitting the interface of the receiver?
Also keep in mind that the `tcp` output will block the rest of the pipeline if the receiver is down or unreachable, causing what you've described (no more events viewable in Kibana).

**Reply by b4b857f6ee** [reply]
Thank you for answering me. Yes, i've read this on another question so i try udp.. nothing :/....

**Comment by b4b857f6ee** [comment]
Other info 
![image](https://user-images.githubusercontent.com/36012515/135231074-ecd4a311-905c-459c-b787-8cbed1623824.png)

**Comment by b4b857f6ee** [comment]
the .2 is the securityonion 
![image](https://user-images.githubusercontent.com/36012515/135232609-4d78a468-065e-4eed-b7c5-1601d90050ef.png)

**Comment by clairmont32** [comment]
are you trying to export only suricata alerts or elastalert? have you tried the config with a module specified? the sample from the docs should work for dns events as a good test but you may need to play around with the config a bit to get the type of events you're looking for

**Reply by b4b857f6ee** [reply]
Thank you for answering me. My goal is to send only the alert for testing purpose :(
